sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -31605,6 +31717,26 @@
             }
           ],
           "id": "emvzoox"
+        },
+        {
+          "input": true,
+          "tableView": false,
+          "key": "dependantstatus",
+          "label": "Dependant Status",
+          "protected": false,
+          "unique": false,
+          "persistent": true,
+          "type": "hidden",
+          "tags": [],
+          "conditional": {
+            "show": "",
+            "when": null,
+            "eq": ""
+          },
+          "properties": {},
+          "calculateServer": true,
+          "calculateValue": "/**\r\n * When parent information is entered if student selects\r\n * both the parents passed away \r\n * or estranged from both of my parents then consider the\r\n * student as not dependant.\r\n */\r\nconst isDependant =\r\n  data.showParentInformation &&\r\n  (data.parentbcResident === \"yes\" || data.parentbcResident === \"no\");\r\n\r\nvalue = isDependant ? \"dependant\" : \"independant\";",
```

Comment:
I think variables with the prefix `is` should be "boolean" accepting `true` and `false` or at most "yes" and "no".

Reply:
I didn't get the question.